bignum_hbls.go

```diff
@@ -0,0 +1,10 @@
+// +build !bignum_pure,!bignum_hol256,!bignum_kilic
```

Comment:
I would just put `!bignum_kilic` here, so that the other two versions (who, incidentally, don't work in go-kzg) and any future version will work here.